cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -56,4 +64,66 @@ describe('Dashboard', () => {
       });
     });
   });
+
+  describe('Textbox', () => {
+    beforeEach(() => {
+      createNewDashboard('Foo Bar');
```

Comment:
In case my comment above actually works it'll be better to use the API here (dashboard creation has already been tested) and to have this in the `before` hook.